spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -405,15 +422,14 @@ definitions:
               Phase biases corrections for a
               satellite being tracked.
 
-
  - MSG_SSR_STEC_CORRECTION:
     id: 0x05EB
     short_desc: Slant Total Electron Content
     desc: |
       The STEC per space vehicle, given as polynomial approximation for
-      a given grid.  This should be combined with SSR-GriddedCorrection
+      a given grid area.  This should be combined with MSG_SSR_GRIDDED_CORRECTION
       message to get the state space representation of the atmospheric
-      delay.
+      delay. It is typically equivalent to the QZSS CLAS Sub Type 8 messages
```

Comment:
?

Reply:
I wanted to put a reference to the CLAS message since you have to use the correction algorithm from that document. This way someone not familiar will know where to go find the details